code/software/dhrystone/Makefile

```diff
@@ -0,0 +1,61 @@
+# Make rosco_m68k dhrystone
+
+# Copyright (c)2020 Ross Bamford
+# See LICENSE
+
+# Hacked by Xark for dhrystone
+
+EXTRA_CFLAGS?=
+SYSINCDIR?=../libs/build/include
+SYSLIBDIR?=../libs/build/lib
+# TODO: Figure out why -lgcc causes: m68k-elf-ld: cannot find -lgcc
+#       Until then, workaround is to point to libgcc.a on your system below. -Xark
```

Comment:
I think this (added to LDFLAGS) will fix this issue:

```
-L $(shell dirname `m68k-elf-gcc $CFLAGS -print-libgcc-file-name`)
```

Reply:
Cool, that makes some sense (as we are calling linker directly, not letting gcc do it).

Reply:
Okay, I have made that change and it seems fine.

Reply:
Let me know if I need to issue another PR or something (I am am Perforce guy at work).

Reply:
Awesome, thanks :) Shouldn't need another PR, I'll rebase this onto develop and merge.